text/0249-v2-experiments.md

```diff
@@ -151,12 +151,14 @@ Disadvantages:
 1. It's not possible for stable modules to depend on unstable ones
   (see Appendix B for data on how necessary that is for the CDK currently).
   This has serious side implications:
-  - All unstable modules that have stable dependents today will have to be
-    graduated before `v2.0` is released.
-  - Before a module is graduated, all of its dependencies need to be graduated.
-  - It will not be possible to add new dependencies on unstable modules to
-    stable modules in the future (for example, that's a common need for
-    StepFunction Tasks).
+
+- All unstable modules that have stable dependents today will have to be
```

Comment:
I believe the intent was to make this a sublist under the numbered.

Doesn't look like it does that - https://github.com/aws/aws-cdk-rfcs/blob/chore/lint-fixes/text/0249-v2-experiments.md#option-2-separate-single-unstable-package

Reply:
Fixed!